tools/scripts/tidy.toml

```diff
@@ -4,6 +4,12 @@
 
 # This is a cargo-make file included in the toplevel Makefile.toml
 
+[tasks.clippy-all]
+description = "Run clippy (this target is used locally, not on CI)"
+category = "ICU4X Development"
+command = "cargo"
+args = ["clippy", "--all-targets", "--all-features", "--", "-D", "warnings", "-Aclippy::unknown-clippy-lints", "-Aclippy::field-reassign-with-default"]
```

Comment:
issue: there should be a comment here saying that this list must be kept in sync with the github action, and a comment there pointing here

Reply:
Actually, no, we already have `cargo clippy-all`, this should just _call_ `cargo clippy-all`

Reply:
Sorry, I should not have mixed the `cargo quick` fix in the same PR.  I moved it to https://github.com/unicode-org/icu4x/pull/826.

Please re-review this PR only from the data perspective.